tests/pmap_test.py

```diff
@@ -1827,6 +1844,21 @@ def f(x):
     self.assertAllClose(pmap(pmap(f, axis_name='j'), axis_name='i')(x),
                         vmap(vmap(f, axis_name='j'), axis_name='i')(x))
 
+  @skipIf(not jax.config.omnistaging_enabled,
```

Comment:
This test seems pretty limited, but I think the same is true for many other collectives' vmap rules. We don't have to act on it in this PR, but we need more systematic tests of collectives, like we have in lax_vmap_test.py.

Reply:
Yeah, it's a pretty unsatisfactory test. As you say, I think it's probably best to address all of these vmap collective tests in a follow-up.